server/handlers/authHandler/schemas.js

```diff
@@ -1,7 +1,7 @@
 const Joi = require('joi');
 
 const authPostSchema = Joi.object({
-  wallet: Joi.string().min(4).max(36).required(),
+  wallet: Joi.alternatives().try(Joi.string(), Joi.string().uuid()).required(),
```

Comment:
we are not going to accept uuid as an option here

Reply:
Currently AuthService uses wallet service's `getByIdOrName` to fetch the wallet. Should that be changed?

Reply:
yeah, we can change it to use just a getByName function